### DISCUSSION TITLE
Welcome to FILE Discussions!

### DISCUSSION THREAD

**Discussion by Hannan-404** [Hannan-404/FILE#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 9kq2** [comment]
Please help keep repeating this without entering the program   [-] Checking For Update...
[✓] 64Bit Found
[+] Loading Modules Please Wait!
[!] Installing Module Pyrebase
Collecting Pyrebase
  Using cached Pyrebase-3.1.0-py3-none-any.whl
Requirement already satisfied: requests>=2.11.0 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from Pyrebase) (2.29.0)
Requirement already satisfied: google-cloud-storage>=1.8.0 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from Pyrebase) (2.8.0)
Requirement already satisfied: google-auth>=1.4.0 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from Pyrebase) (2.17.3)
Requirement already satisfied: requests-toolbelt>=0.7.0 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from Pyrebase) (1.0.0)
Requirement already satisfied: cachetools<6.0,>=2.0.0 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from google-auth>=1.4.0->Pyrebase) (5.3.0)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from google-auth>=1.4.0->Pyrebase) (0.3.0)
Requirement already satisfied: six>=1.9.0 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from google-auth>=1.4.0->Pyrebase) (1.16.0)
Requirement already satisfied: rsa<5,>=3.1.4 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from google-auth>=1.4.0->Pyrebase) (4.9)
Requirement already satisfied: google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from google-cloud-storage>=1.8.0->Pyrebase) (2.11.0)
Requirement already satisfied: google-cloud-core<3.0dev,>=2.3.0 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from google-cloud-storage>=1.8.0->Pyrebase) (2.3.2)
Requirement already satisfied: google-resumable-media>=2.3.2 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from google-cloud-storage>=1.8.0->Pyrebase) (2.5.0)
Requirement already satisfied: charset-normalizer<4,>=2 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from requests>=2.11.0->Pyrebase) (3.1.0)
Requirement already satisfied: idna<4,>=2.5 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from requests>=2.11.0->Pyrebase) (3.4)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from requests>=2.11.0->Pyrebase) (1.26.15)
Requirement already satisfied: certifi>=2017.4.17 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from requests>=2.11.0->Pyrebase) (2022.12.7)
Requirement already satisfied: googleapis-common-protos<2.0dev,>=1.56.2 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-cloud-storage>=1.8.0->Pyrebase) (1.59.0)
Requirement already satisfied: protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-cloud-storage>=1.8.0->Pyrebase) (4.22.3)
Requirement already satisfied: google-crc32c<2.0dev,>=1.0 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from google-resumable-media>=2.3.2->google-cloud-storage>=1.8.0->Pyrebase) (1.5.0)
Requirement already satisfied: pyasn1<0.6.0,>=0.4.6 in /data/data/com.termux/files/usr/lib/python3.11/site-packages (from pyasn1-modules>=0.2.1->google-auth>=1.4.0->Pyrebase) (0.5.0)

**Comment by danishhacker420** [comment]
Bro login krse krna h ham new h nahi pata mujhe

**Comment by Kaifsiddiqui7890** [comment]
Bhai yah cloning Nahin chalu ho raha batao na Kaise chalu hota hai

**Comment by Bacim2008** [comment]
Oh brother Is there Phishing file And a tool

**Comment by darkdevil120** [comment]
This tool is working but it create the less txt file of the id. Old works better then this. Please upload the old script not 5his this one works not better
